x-pack/legacy/plugins/siem/public/components/timeline/body/events/stateful_event.tsx

```diff
@@ -141,6 +142,8 @@ const StatefulEventComponent: React.FC<Props> = ({
 
   const divElement = useRef<HTMLDivElement | null>(null);
 
+  const eventType = useMemo(() => getEventType(event.ecs), [event]);
```

Comment:
Isn't eventType just a string? We shouldn't need to memoize strings or any primitive anywhere for that matter I would imagine.